runelite-client/src/main/java/net/runelite/client/plugins/chatchannel/ChatChannelPlugin.java

```diff
@@ -341,6 +346,11 @@ public void onGameTick(GameTick gameTick)
 			}
 		}
 
+		if (config.clanChatShowOnlineMemberCount())
```

Comment:
Typically we prefer to put these into script events if possible, like how the friends chat channel counter works. Have you tried that?

Reply:
I'll try to do that, thanks for the tip!